Mathlib/CategoryTheory/Monoidal/Free/Coherence.lean

```diff
@@ -82,9 +82,7 @@ theorem inclusion_obj (X : N C) :
 @[simp]
 theorem inclusion_map {X Y : N C} (f : X ⟶ Y) :
     inclusion.map f = eqToHom (congr_arg _ (Discrete.ext (Discrete.eq_of_hom f))) := by
-  rcases f with ⟨⟨⟩⟩
-  cases Discrete.ext (by assumption)
-  apply inclusion.map_id
+  rfl
```

Comment:
This should be `rfl`, not `by rfl`.

Reply:
If you remove the `rfl` on line 276, that should fix the error.

Reply:
bors d+